src/main/java/com/project/ita5/school/SchoolServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.project.ita5.school;
+
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.text.Collator;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Service
+public class SchoolServiceImpl implements SchoolService {
+    @Override
+    public List<String> fetchSchools() {
+        RestTemplate restTemplate = new RestTemplate();
+        String url1 = "https://lt.wikipedia.org/w/api.php?action=parse&format=json&prop=sections&page=Sąrašas:Lietuvos_aukštosios_mokyklos&section=1&prop=wikitext";
```

Comment:
Such strings should not be hardcoded in the code. Use resources files for those.

Reply:
Done